components/tensorflow-notebook-image/Dockerfile

```diff
@@ -59,6 +47,13 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
   && apt-get clean && \
   rm -rf /var/lib/apt/lists/*
 
+# Install Nodejs for jupyterlab-manager
+RUN curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+RUN apt-get update && apt-get install -yq --no-install-recommends \
```

Comment:
It looks like the E2E builds are failing because pip isn't in the image so you might want to install it here.

Reply:
I have added installation for `python3-setuptools` and `python3-pip`.